### PR TITLE
Fix data source vertical field size

### LINKED ISSUE #2904: Table expansion- national data & tab 5c
@minotogna 
**Is your feature request related to a problem? Please describe.**
Does not expand the cell(s)(as below) which suits for the text length:
1. National data:

2. Tab 5c:

**Describe the solution you'd like**
All cells in the same row should automatically adjust to a suitable size, based on the largest cell in that row which should fully contain the content (text).

**Priority**
Low

### PR DESCRIPTION
**Pull request by sorja**
firefox_KexXE4XTlI

firefox_KCyNKSfjP7

### CODE REVIEW

**Comment by minotogna** [comment]
The first video shows text aligned to the right ? Can't this be fixed ?

**Comment by sorja** [comment]
> The first video shows text aligned to the right ? Can't this be fixed ?

Sure

image

**Comment by minotogna** [comment]
@sorja can we have a new video?

File: src/client/pages/Section/DataTable/Table/Row/RowData/Cell/Cell.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .fra-table__select-container {
   &.multiple {
     overflow: visible;
+    text-align: left;
```
Comment (minotogna) [inline src/client/pages/Section/DataTable/Table/Row/RowData/Cell/Cell.scss:4]: does in ar language is aligned to the right?

Comment (sorja) [reply src/client/pages/Section/DataTable/Table/Row/RowData/Cell/Cell.scss:4]: firefox_0PerPWEvHT

Comment (sorja) [reply src/client/pages/Section/DataTable/Table/Row/RowData/Cell/Cell.scss:4]: firefox_64SyeNal30

**Review by minotogna — APPROVED** [review]

**Pull request merged**